articles/evaluation_queues.md

```diff
@@ -174,11 +177,13 @@ Select **Submit To Challenge** to pick the challenge for your submission. Follow
 
 <img id="toobig" src="/assets/images/submit_file_to_challenge.png">
 
+## View Submissions of an Evaluation Queue
 
+### Viewing Submissions
 
-## View Submissions of an Evaluation Queue
+All Submissions to an Evaluation queue can be viewed through a leaderboard.  Annotations can be added to a SubmissionStatus to be displayed.  Each of these added annotations can be set to either public or private.  Private annotations cannot be read by people on the a leaderboard unless the Team or Synapse user has **Can Score** or **Admin** permissions on the Evaluation queue.  Public annotations can be viewed by any Team or user that have **Can View** permissions.
```

Comment:
Do evaluation queue annotations behave differently than synapse annotations? (Since annotations cannot be private). If yes, is there another term we can use to avoid confusion? 

I recommend we provide an example of how to add annotations to display the SubmissionStatus.